File: Week 3/C++/AMR15A.cpp
#include <iostream>
using namespace std;

int main() {
	int n, x, count = 0;
    
    cin >> n;
	for(int i = 0; i < n; ++i) {
	    cin >> x;
	    if (x % 2 == 0)
	        count++;
	}
	if (count > n-count)
	    cout << "READY FOR BATTLE" << endl;
	else
	    cout << "NOT READY" << endl;
	    
	return 0;
}

File: Week 3/Java/AMR15A.java
import java.util.*;
import java.lang.*;
import java.io.*;

class Codechef
{
	public static void main (String[] args) throws java.lang.Exception
	{
		int ar[];
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        ar = new int[n];
        for(int i = 0; i < n; i++) {
            ar[i] = sc.nextInt();
        }
        int count = 0;
        for(int j = 0; j < n; j++) {
            if(ar[j] % 2 == 0) {
                count++;
            }
        }
        if(count > n / 2) {
            System.out.println("READY FOR BATTLE");
        } else {
            System.out.println("NOT READY");
        }
	}
}

